product_variant_configurator/__manifest__.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2015 Oihane Crucelaegui (AvanzOSC)
+# Copyright 2016 ACSONE SA/NV
+# Copyright 2017 David Vidal <david.vidal@tecnativa.com>
+# Copyright 2015-2017 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Product Variant Configurator',
+    'summary': """
+        Provides an abstract model for product variant configuration.""",
+    'version': '11.0.1.0.0',
+    'category': 'Product Variant',
+    'license': 'AGPL-3',
+    'author': 'AvanzOSC,'
+              'Tecnativa,'
+              'ACSONE SA/NV,'
+              'Odoo Community Association (OCA)',
```

Comment:
Add a space after each comma that separates the names of the companies. Eg: 'AvanzOSC, '

Reply:
This is not needed at all